src/components/ChataTable/ChataTable.js

```diff
@@ -103,13 +103,18 @@ export default class ChataTable extends React.Component {
       this.tableOptions.filterMode = this.useRemote // v4: ajaxFiltering = true
       this.tableOptions.pagination = false
       this.tableOptions.paginationMode = this.useRemote
-      this.tableOptions.progressiveLoad = 'scroll' // v4: ajaxProgressiveLoad
-      this.tableOptions.ajaxURL = 'https://required-placeholder-url.com'
       this.tableOptions.paginationSize = this.pageSize
       this.tableOptions.paginationInitialPage = 1
-      this.tableOptions.ajaxRequesting = (url, params) => this.ajaxRequesting(props, params)
-      this.tableOptions.ajaxRequestFunc = (url, config, params) => this.ajaxRequestFunc(props, params)
-      this.tableOptions.ajaxResponse = (url, params, response) => this.ajaxResponseFunc(props, response)
+      if (this.useRemote === 'local') {
```

Comment:
Just a heads up, when using string values like 'local', we should check to see how many times this value is used and if it 3 or more we should create a constant string value for. I checked for 'local' and it looks like it is only being used three times with the updates to this PR. 

For reference: https://next.sonarqube.com/sonarqube/coding_rules?open=java%3AS1192&rule_key=java%3AS1192